app/src/androidTest/java/ch/epfl/sweng/SDP/game/WaitingPageActivityTest.java

```diff
@@ -48,6 +57,9 @@
 
     private DataSnapshot dataSnapshotMock;
     private DatabaseError databaseErrorMock;
+    private Intent intentMock;
+    private ConnectivityManager cmMock;
+    private Context spyContext;
```

Comment:
I agree from cmMock maybe. But not for spyContext, I mean it's not a mock but we spy directly from the instance of the context. So instead of calling it mockContext it's just spyContext. We keep the same syntax as the above member variables.